src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -60,17 +60,45 @@ public void TestGetDrives()
         }
 
         [Fact]
+        public void TestDriveProperties_AppContainer()
+        {
+            DriveInfo validDrive = DriveInfo.GetDrives().Where(d => d.DriveType == DriveType.Fixed).First();
+            bool isReady = validDrive.IsReady;
+            Assert.NotNull(validDrive.Name);
+            Assert.NotNull(validDrive.RootDirectory.Name);
+
+            if (PlatformDetection.IsWinRT)
+            {
+                Assert.Throws<UnauthorizedAccessException>(() => validDrive.AvailableFreeSpace);
+                Assert.Throws<UnauthorizedAccessException>(() => validDrive.DriveFormat);
+                Assert.Throws<UnauthorizedAccessException>(() => validDrive.TotalFreeSpace);
+                Assert.Throws<UnauthorizedAccessException>(() => validDrive.TotalSize);
+                Assert.Throws<UnauthorizedAccessException>(() => validDrive.VolumeLabel);
+            }
+            else
+            {
+                Assert.NotNull(validDrive.DriveFormat);
+                Assert.True(validDrive.AvailableFreeSpace > 0);
+                Assert.True(validDrive.TotalFreeSpace > 0);
+                Assert.True(validDrive.TotalSize > 0);
+                Assert.NotNull(validDrive.VolumeLabel);
```

Comment:
some of the properties are being tested in a test method below but only if validDrive IsReady and not inside an AppContainer. I would keep that method.